packages/jbrowse-web/src/plugins/LinearGenomeView/models/index.js

```diff
@@ -102,7 +100,22 @@ export default function LinearGenomeViewStateFactory(pluginManager) {
         return getConf(self, 'reversed')
       },
     }))
+    .volatile(() => ({
+      displayedRegions: [],
+    }))
     .actions(self => ({
+      afterAttach() {
+        const displayedRegionsDisposer = autorun(() => {
+          const { assemblyManager } = getRoot(self)
+          if (assemblyManager)
+            self.setDisplayedRegions(assemblyManager.displayedRegions)
+        })
+
```

Comment:
This is very problematic, because it means that all linear genome views are forced to show the same set of regions now. We can merge this in the interest of avoiding long-lived branches, but it will need to change very soon.

Reply:
I think if we just move this "binding" of the view to an assembly to the root model, that would remove most of my concern.

Reply:
Agreed, there needs to be something else that manages the displayed regions. Like I mentioned on gitter, maybe a RegionManager? It needs to be able to pull regions from several potential sources (assembly, config, regions file, user interaction, etc.) and keep track of which one is being used.

Reply:
Maybe a ViewManager, which would be where we will put our logic for managing what views are shown, what kind of views they are, what regions they are showing, etc.

Because we are going to have a lot more kinds of views besides just LinearGenomeViews, and they will each